tests/persist.test.js

```diff
@@ -1114,51 +1114,51 @@ test("multiple changes don't cause concurrent persist operations", async () => {
     counter: 1,
   });
 
-  await wait(20);
+  await wait(17);
 
   expect(memoryStorage.store['[EasyPeasyStore][0]']).toBeUndefined();
 
   store.getActions().change({
     counter: 2,
   });
 
-  await wait(20);
+  await wait(17);
 
   expect(memoryStorage.store['[EasyPeasyStore][0]']).toBeUndefined();
 
   store.getActions().change({
     counter: 3,
   });
 
-  await wait(20);
+  await wait(17);
 
   expect(memoryStorage.store['[EasyPeasyStore][0]']).toBeUndefined();
 
   store.getActions().change({
     counter: 4,
   });
 
-  await wait(20);
+  await wait(16);
 
   expect(memoryStorage.store['[EasyPeasyStore][0]']).toBeUndefined();
 
   store.getActions().change({
     counter: 5,
   });
 
-  await wait(20);
+  await wait(17);
```

Comment:
Not ideal, but it works on my machine 🤷

Reply:
[it didn't work ](https://github.com/ctrlplusb/easy-peasy/actions/runs/3096631134/jobs/5012402806#step:4:62)